.eslintrc.cjs

```diff
@@ -47,6 +64,48 @@ module.exports = {
 				"@typescript-eslint/no-unsafe-call": "off",
 			},
 		},
+		{
+			files: ["**/*.{yml,yaml}"],
+			parser: "yaml-eslint-parser",
+			extends: ["plugin:yml/base", "plugin:yml/prettier"],
+			rules: {
+				"yml/sort-keys": [
+					"error",
+					{
+						order: [
+							// github workflow root-level keys
+							"name",
+							"on",
+							"permissions",
+							"jobs",
+							// github issue template
+							"description",
+							"title",
+							"labels",
+							"body",
+							// github job
+							"runs-on",
+							"using",
+							"steps",
+							{
+								order: {
+									natural: false,
+									type: "asc",
+								},
+							},
+						],
```

Comment:
I would actually rather just get rid of all overrides and be in alphabetical order altogether. Simpler configuration and fewer personal preferences to remember.